### PR TITLE
Newsletter 263 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2023-08-09-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+[news245 blinded]: /fr/newsletters/2023/04/05/#bolts-765
+[towns dos]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004020.html
+[news86 hold fees]: /en/newsletters/2020/02/26/#reverse-up-front-payments
+[shikhelman dos]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004033.html
+[towns dos2]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004035.html
+[kcs endorsement]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004034.html
+[todd rbf]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-July/021823.html
+[towns rbf]: https://github.com/bitcoin/bitcoin/pull/28132#issuecomment-1657669845
+[news207 onion]: /en/newsletters/2022/07/06/#onion-message-rate-limiting
+[news261 jamming]: /en/newsletters/2023/07/26/#channel-jamming-mitigation-proposals
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-08-09-newsletter.md:287]: I think:

```suggestion
[news261 jamming]: /fr/newsletters/2023/07/26/#propositions-d-attenuation-des-attaques-de-saturation-des-canaux
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

File: _posts/fr/newsletters/2023-08-09-newsletter.md (hunk 1 above)
Comment (bitschmidty) [reply _posts/fr/newsletters/2023-08-09-newsletter.md]: @Copinmalin I applied this suggestion to the code. Let me know if you think this PR is good to go

**Review by murchandamus — APPROVED** [review]
LGTM, but I’m not @Copinmalin ;)

**Comment by Copinmalin** [comment]
you are my guides, if you agree i agree 😅

**Comment by murchandamus** [comment]
Okay, shipped it. :)

**Pull request merged**